src/lib/stepper/stepper.spec.ts

```diff
@@ -0,0 +1,767 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {Component, DebugElement} from '@angular/core';
+import {MdStepperModule} from './index';
+import {By} from '@angular/platform-browser';
+import {MdHorizontalStepper} from './stepper-horizontal';
+import {MdVerticalStepper} from './stepper-vertical';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {FormControl, FormGroup, ReactiveFormsModule, Validators} from '@angular/forms';
+import {MdStepperNext, MdStepperPrevious} from './stepper-button';
+import {dispatchKeyboardEvent} from '@angular/cdk/testing';
+import {ENTER, LEFT_ARROW, RIGHT_ARROW, SPACE} from '@angular/cdk/keycodes';
+import {MdStepper} from './stepper';
+
+const EMAIL_REGEX = /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
Since the unit test doesn't really care about what you're validating, you can use a simpler regex that only validates a certain value, e.g. `const UNICORN_REGEX = /unicorn/`.